packages/dapp/src/components/listing/ChallengeDetail.tsx

```diff
@@ -397,14 +453,14 @@ class ChallengeDetail extends React.Component<ChallengeDetailProps, ChallengeVot
   };
 
   private approveVotingRights = async (): Promise<TwoStepEthTransaction<any> | void> => {
-    const numTokens: BigNumber = new BigNumber(this.state.numTokens as string).mul(1e18);
+    const numTokens: BigNumber = new BigNumber((this.state.numTokens as string).replace(",", "")).mul(1e18);
```

Comment:
long term might be better to prevent them from typing commas in the first place? probably a good react component out there already for this